core/controller/src/main/scala/whisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -291,7 +291,9 @@ class ShardingContainerPoolBalancer(config: WhiskConfig, controllerInstance: Con
     val start = transid.started(
       this,
       LoggingMarkers.CONTROLLER_KAFKA,
-      s"posting topic '$topic' with activation id '${msg.activationId}'",
+      invoker.hostname
+        .map(h => s"posting topic '$topic' to invoker '$h' with activation id '${msg.activationId}'")
+        .getOrElse(s"posting topic '$topic' with activation id '${msg.activationId}'"),
```

Comment:
Could we also print the whole instanceId?

As in: `s"posting to $invoker using topic '$topic' with activation id '${msg.activationId}'"`

Should then log the whole case class with its respective members.